src/main/java/com/google/codeu/data/Message.java

```diff
@@ -26,7 +26,8 @@
   private long timestamp;
   private String text;
   private String imageUrl;
-  private String imageLabels;
+  // private String imageLabels;
```

Comment:
Is this commented out for future reference?

Reply:
It was for the previous stage of development. Will remove!